### PR TITLE
User Guide updates

### PR DESCRIPTION
**Pull request by amrut-prabhu**
Touched up user guide and added descriptions of new features
Issue #70: Updated Team

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -10,14 +11,25 @@
...
+*Club Connect* is an application targeted at <<DeveloperGuide#club,club>> <<DeveloperGuide#member,members>> who are students at the National University of Singapore (NUS). It aims to make the tedious process of club management easier are more effective.
+
+Currently, Club Connect is a desktop app that makes use of a *Command Line Interface* (<<DeveloperGuide#cli,CLI>>) for increased productivity, while still having the benefits of a Graphical User Interface (GUI). So, you can have increased productivity (owing to the CLI) without having to give up the beauty and aesthetics of a traditional GUI application.
+
+The faster you type, the faster Club Connect gets your club management tasks done. Interested? Continue on to <<Quick Start>> to get started. Enjoy!
```
Comment (yash-chowdhary) [inline docs/UserGuide.adoc]: Line 26: club-management*

Comment (yash-chowdhary) [inline docs/UserGuide.adoc]: Line 26: club-management*

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -27,92 +39,98 @@ AddressBook Level 4 (AB4) is for those who *prefer to use a desktop app for mana
...
 
 ====
 *Command Format*
 
-* Words in `UPPER_CASE` are the parameters to be supplied by the user e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
-* Items in square brackets are optional e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
-* Items with `…`​ after them can be used multiple times including zero times e.g. `[t/TAG]...` can be used as `{nbsp}` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
+* Words in `UPPER_CASE` are the parameters to be supplied by the user. For example, in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe` or `add n/Jane Doe`.
+* Items in square brackets are optional e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/President` or as `n/John Doe`.
+* Items with `…`​ after them can be used multiple times including zero times e.g. `[t/TAG]...` can be used as `{nbsp}` (i.e. 0 times), `t/Treasurer`, `t/Operations t/Logistics` etc.
```
Comment (yash-chowdhary) [inline docs/UserGuide.adoc]: Logistics is not a tag, it's a group. Maybe remove it or change it to something else?

**Review by yash-chowdhary — COMMENTED** [review]
some minor changes required

File: docs/UserGuide.adoc (hunk 2 above)
Comment (amrut-prabhu) [reply docs/UserGuide.adoc]: Fixed

File: docs/UserGuide.adoc (hunk 1 above)
Comment (amrut-prabhu) [reply docs/UserGuide.adoc]: Fixed

**Pull request merged**